Signal/translations/en.lproj/Localizable.strings

```diff
@@ -518,7 +518,7 @@
 "FAILED_SENDING_BECAUSE_RATE_LIMIT" = "Too many failures with this contact. Please try again shortly.";
 
 /* action sheet header when re-sending message which failed because of untrusted identity keys */
-"FAILED_SENDING_BECAUSE_UNTRUSTED_IDENTITY_KEY" = "You must accept the new safety number before you will be able to send.";
+"FAILED_SENDING_BECAUSE_UNTRUSTED_IDENTITY_KEY" = "Recipient's safety number recently changed. You may wish to verify before resending.";
```

Comment:
I've updated this to "Your safety number with %@ has recently changed. You may wish to verify before resending."